threema/gateway/e2e.py

```diff
@@ -659,10 +659,15 @@ class DeliveryReceipt(Message):
     """
     @enum.unique
     class ReceiptType(enum.IntEnum):
-        """Describes message receipt types."""
+        """
+        Describes message receipt types.
+
+        .. warning:: `user_ack` is deprecated and will be removed with
+           the next major release. Use `user_acknowledge` instead.
```

Comment:
Ah, then never mind.

An alternative would be to override `__getattribute__` to show the warning, but that's probably overkill :)